src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php

```diff
@@ -51,28 +52,28 @@ public function guessType($class, $property)
         }
 
         switch ($metadata->getTypeOfField($property)) {
-            case 'array':
+            case Type::TARRAY:
                 return new TypeGuess('collection', array(), Guess::MEDIUM_CONFIDENCE);
-            case 'boolean':
+            case Type::BOOLEAN:
                 return new TypeGuess('checkbox', array(), Guess::HIGH_CONFIDENCE);
-            case 'datetime':
+            case Type::DATETIME:
+            case Type::DATETIMETZ:
             case 'vardatetime':
```

Comment:
Not sure why this is here. There doesn't seem to be corresponding type in Doctrine.